pytest/tests/loadtest/locust/common/base.py

```diff
@@ -862,15 +862,12 @@ def do_on_locust_init(environment):
         num_funding_accounts = environment.parsed_options.max_workers
         funding_balance = 100000 * NearUser.INIT_BALANCE
 
-        def create_account(id):
-            account_id = f"funds_worker_{id}.{master_funding_account.key.account_id}"
-            return Account(key.Key.from_seed_testonly(account_id))
-
-        funding_accounts = [
-            create_account(id) for id in range(num_funding_accounts)
-        ]
-        node.prepare_accounts(funding_accounts, master_funding_account,
-                              funding_balance, "create funding account")
+        for index in range(num_funding_accounts):
+            account_id = f"funds_worker_{index}.{master_funding_account.key.account_id}"
+            account = Account(key.Key.from_seed_testonly(account_id))
+            node.prepare_account(account, master_funding_account,
```

Comment:
how does this provide better error messages? do we somehow lose the errors when running these in parallel?

Reply:
Yes, see the explanation here: https://github.com/near/nearcore/blob/9a41e3ec52ccfb8f5f5ad50d2978c56b6f7ab234/pytest/tests/loadtest/locust/common/base.py#L465

In short, `prepare_accounts` uses async API to submit transactions, which does not propagate errors if transactions fail initial validation. I haven't found a way to fix that behavior while keeping the performance benefits, so I'm using `prepare_account` that uses a sync API to submit transactions under the hood.